modules/simple_forms_api/spec/requests/v1/uploads_spec.rb

```diff
@@ -203,10 +203,12 @@ def self.test_failed_request_scrubs_error_message_unhandled_form
 
         expect(response).to have_http_status(:error)
         expect(response.body).to include('something has gone wrong with your form')
-        expect(response.body).not_to include(data.dig('veteran', 'ssn')&.[](0..2))
-        expect(response.body).not_to include(data.dig('veteran', 'ssn')&.[](3..4))
-        expect(response.body).not_to include(data.dig('veteran', 'ssn')&.[](5..8))
-        expect(response.body).not_to include(data.dig('veteran', 'address', 'postal_code')&.[](0..4))
+
+        exception = JSON.parse(response.body)['errors'][0]['meta']['exception']
```

Comment:
We changed the approach in this spec because pieces of the ssn matched line numbers in the stack trace. So rather than searching the body, that included both the error and stack trace, we parsed out the exception and used that in the expectation instead.